seed/tests/test_column_views.py

```diff
@@ -152,6 +152,28 @@ def test_get_all_columns(self):
         # randomly check a column
         self.assertIn(expected, data)
 
+    # tests that units get added to columns
+    def test_column_units(self):
+
+        # get the columns list without units
+        responseWithoutUnits = self.client.get(reverse('api:v3:columns-list'), {
+            'organization_id': self.org.id,
+            'display_units': 'false'
+        })
+
+        # get an entry with EUI units
+        columnWithoutUnits = next((x for x in json.loads(responseWithoutUnits.content)['columns'] if x['data_type'] == 'eui'), None)
```

Comment:
In this case, since the set of Column objects that are created on initial org creation is well defined, I'd vote to try to isolate a specific Column object (vs the first one that returns a 'data_type' of 'eui'). That way you can assert against a string (`assertEqual(..., "Site EUI")` in both the with and without cases.

IMO, that would add clarity and consistency to these tests.

Reply:
I've now selected a specific record and am testing off of that.